pkg/apply/solver/solver_test.go

```diff
@@ -429,6 +429,12 @@ func TestTaskQueueBuilder_AppendApplyWaitTasks(t *testing.T) {
 							expTsk.Ids, actWaitTask.Ids)
 					}
 					assert.Equal(t, taskrunner.AllCurrent, actWaitTask.Condition)
+					if tc.options.ReconcileTimeout != 0 {
```

Comment:
Can we just replace this entire for loop with `testutil.AssertEqual(t, tc.expectedTasks, tq.tasks)`?